include/oneapi/dpl/pstl/hetero/algorithm_ranges_impl_hetero.h

```diff
@@ -547,12 +705,26 @@ __pattern_count(__hetero_tag<_BackendTag>, _ExecutionPolicy&& __exec, _Range&& _
 }
 
 #if _ONEDPL_CPP20_RANGES_PRESENT
+
+template <typename _Pred, typename _Proj>
+struct __pattern_count_if_pred
```

Comment:
We already defined a type with the same functionality above. And we can re-use one.

Reply:
Fixed